src/Rector/Composer/ExtensionComposerRector.php

```diff
@@ -17,7 +17,7 @@
  * @changelog https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ExtensionArchitecture/ComposerJson/Index.html#extra
  * @see \Ssch\TYPO3Rector\Tests\Rector\Composer\ExtensionComposerRector\ExtensionComposerRectorTest
  */
-final class ExtensionComposerRector implements ExtensionComposerRectorInterface
+final class ExtensionComposerRector implements ComposerRectorInterface
```

Comment:
Exactly same behavior is supported in Rector natively with this interface :+1:

Reply:
Yes and no. I wanted to separate these to not have any side effect. Please don´t change it.

Reply:
Wait. I am gonna think about it twice.

Reply:
It's 95 % of duplicated code from Rector internals with some prefixed classes. This API is not public and is already covered by Rector.

What exact side effects? We should fix those if there are some troubles

Reply:
I guess you are right. There were a reason historically. But i think it should be fine now as we have FileProcessors in place. Can´t  remember exactly. So seems good for now.

Reply:
If there are any bugs, let me know. It's definitely wrong in Rector :)

Reply:
I think it should be fine now. There was a time with the composer.json be part of ProcessCommand deep inside. This is gone now.